extension/background.js

```diff
@@ -1,5 +1,5 @@
 const googleSsoRegex = /name="SAMLResponse" value="([\s\S]+?)"/i;
-const accountSelectionRegex = `tabindex="\\d" jsname="\\S\*" data-authuser="(-?\\d)" data-identifier="(\\S\*@DOMAIN)"`;
```

Comment:
I tried this, but it's not consistent, sometimes it works, sometimes the accounts are in the wrong order. It seems unrelated to the `authuser=` query parameter that is sent to get the SAML response below. Iterating is for now the most foolproof approach.